components/pkg-export-kubernetes/src/main.rs

```diff
@@ -95,65 +154,138 @@ fn start(_ui: &mut UI) -> result::Result<(), String> {
+                 operator must be deployed within the Kubernetes cluster to intercept the created \
+                 objects.";
+
+    let app = Cli::new(name, about)
+        .add_base_packages_args()
+        .add_builder_args()
+        .add_tagging_args()
+        .add_publishing_args()
+        .app;
+
+    app.arg(
+        Arg::with_name("OUTPUT")
+            .value_name("OUTPUT")
+            .long("output")
+            .short("o")
+            .help(
+                "Name of manifest file to create. Pass '-' for stdout (default: -)",
+            ),
+    ).arg(
+            Arg::with_name("COUNT")
+                .value_name("COUNT")
+                .long("count")
+                .validator(valid_natural_number)
+                .help("Count is the number of desired instances"),
+        )
+        .arg(
+            Arg::with_name("TOPOLOGY")
+                .value_name("TOPOLOGY")
+                .long("topology")
+                .short("t")
+                .possible_values(&["standalone", "leader"])
+                .help(
+                    "A topology describes the intended relationship between peers \
+                    within a service group. Specify either standalone or leader \
+                    topology (default: standalone)",
+                ),
+        )
+        .arg(
+            Arg::with_name("GROUP")
+                .value_name("GROUP")
+                .long("service-group")
+                .short("g")
+                .help(
+                    "group is a logical grouping of services with the same package and \
+                    topology type connected together in a ring (default: default)",
+                ),
+        )
+        .arg(
+            Arg::with_name("CONFIG_SECRET_NAME")
+                .value_name("CONFIG_SECRET_NAME")
+                .long("config-secret-name")
+                .short("n")
+                .help(
+                    "name of the Kubernetes Secret containing the config file - \
+                    user.toml - that the user has previously created. Habitat will \
+                    use it for initial configuration of the service",
+                ),
+        )
+        .arg(
+            Arg::with_name("RING_SECRET_NAME")
+                .value_name("RING_SECRET_NAME")
+                .long("ring-secret-name")
+                .short("r")
+                .help(
+                    "name of the Kubernetes Secret that contains the ring key, which \
+                    encrypts the communication between Habitat supervisors",
+                ),
+        )
+        .arg(
+            Arg::with_name("BIND")
+                .value_name("BIND")
+                .long("bind")
+                .short("b")
+                .multiple(true)
+                .number_of_values(1)
+                .help(
+                    "Bind to another service to form a producer/consumer relationship, \
+                    specified as name:service:group",
```

Comment:
This should be `<NAME>:<SERVICE GROUP>`, right? That is two parts, not three parts?

Reply:
In the official habitat documentation it states bind expects the following format: `--bind database:mysql.default`, so 3 parts. Unless something changed and documentation wasn't updated?

We then in the Habitat operator convert it to the following [format](https://github.com/kinvolk/habitat-operator/blob/master/docs/api.md#bind) and an [example](https://github.com/kinvolk/habitat-operator/blob/master/examples/bind/habitat.yml#L20-L26) of that.

So we require all three pieces of information to correctly pass that bind information when starting Habitat service. So this from what I can tell is correct.

Reply:
@christophermaier what @LiliC said but if you like the format to be consistent, we can change to "name:service.group".

Reply:
@zeenix @LiliC Sorry, I wasn't clear... I was getting at the fact that the two `:` characters made it look like a three-part value. `name:service.group` works.